src/Tooltip/Tooltip.stories.tsx

```diff
@@ -30,9 +18,22 @@ storiesOf('molecules|Tooltip', module)
       <div>
         <Tooltip
           placement={TooltipPlacement.TOP}
```

Comment:
Is there a reason to specify placement explicitly here? Just curious if there's a default prop value

Reply:
It looks logical for me. But, for sure I can make additional prop

Reply:
Sorry I wasn't clear. What I mean is, is there a difference between

```jsx
<Tooltip placement={TooltipPlacement.TOP} content={...} />
```

and 

```jsx
<Tooltip content={...} />
```

Reply:
Oups, I misunderstood the thing. `TOP` is default, I used here just to be explicit.